atomic-structure/content.md

```diff
@@ -0,0 +1,25 @@
+# Radioactive Decay
+#### Learning objective
+Basic understanding of atomic structure
```

Comment:
Yes, definitely should be in the guide. Perhaps we should pick a preferred form of learning objective. Lots of people like the imperative tense "Reproduce a shell model of the atom." or "Describe the shell model of the atom."